docs/cli/testkube_install.md

```diff
@@ -1,36 +1,32 @@
-# Testkube Install
+## testkube install
 
-Install the Helm chart registry in the current kubectl context.
-
-## **Synopsis**
```

Comment:
Is this removed for a reason?

Reply:
I think I've removed it to make it consistent with other commands - it's quite obvious that you can use flags below ;)